Update README.md

MortenSo opened the pull request:
Hi - I have been tinkering with your SmarterPrecondition project. Actually needed to make a few changes to the code to make it work. First - to install I found I needed to use node instead of nodejs as the command to launch the program. On OSX. New to GitHib, I will tell you about the other changes I made as well if this message gets through :-) Really helpful project!

MortenSo committed:
Update README.md

sveip commented:
I'm curious, what version of node and OS you are running? (Vi er alle norske her, men kanskje greit å ta det "interantional" :) )

MortenSo committed:
Update calendar.js

MortenSo commented:
Running OS X El Capitan (newest on mac) and Node v4.2.1

Commit:
Remove issue from not reading temp in description

Error message from trying to read temp from event description. “Fixed”
by setting temp to 20 degrees permanent

sveip commented:
I would suggest mentioning `n` in the readme, that way `node` vs `nodejs` is not relevant. https://github.com/tj/n

Commit:
Changed trigger on fan_status

!== null doesn’t work when fan_status = 0

Commit:
Minor cleanup
